src/WarcraftLegacies.Source/Setup/Spells/ElementalSpellSetup.cs

```diff
@@ -0,0 +1,37 @@
+﻿using MacroTools.PassiveAbilities;
+using MacroTools.PassiveAbilitySystem;
+using MacroTools.Spells;
+using MacroTools.SpellSystem;
+using WarcraftLegacies.Source.Spells;
+
+namespace WarcraftLegacies.Source.Setup.Spells
+{
+  public static class ElementalSpellSetup
+  {
+    /// <summary>
+    /// Sets up <see cref="ElementalSpellSetup"/>.
+    /// </summary>
+    public static void Setup()
```

Comment:
Could you move all this registration to `Skywall.cs`? Take a look at `Ahnqiraj`'s `RegisterSpells` method for how to do that. It's a new pattern that will save us from registering spells that can't be accessed in a game.